### PR TITLE
add reviews/review-id/comments + testing + error handling

### PR DESCRIPTION
**Pull request by xViolaine**
Hello! Finished task 9 in here

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -124,115 +124,167 @@ describe("GET /api/reviews/:review_id", () => {
...
-                    designer: 'Leslie Scott',
-                    owner: 'philippaclaire9',
-                    review_img_url:
-                        'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
-                    review_body: 'Fiddly fun for all the family',
-                    category: 'dexterity',
-                    created_at: new Date(1610964101251).toISOString(),
-                    votes: 25
+                const { comments } = body;
+                expect(!!comments.length).toEqual(true)
```
Comment (elijaharley) [inline __tests__/app.test.js:188]: Is this just to be extra expressive !!??

File: __tests__/app.test.js [hunk 2]
```diff
@@ -124,115 +124,167 @@ describe("GET /api/reviews/:review_id", () => {
...
-        .patch('/api/reviews/3')
-        .expect(400)
-        .send(newVote)
-        .then(({ body}) => {
-            expect(body.msg).toBe("'string' is not a valid value!")
-        })
-    });
-})
-
+})
```
Comment (elijaharley) [inline __tests__/app.test.js]: I'd personally be inclined to have all of the error handling for each endpoint below each endpoint as this means not having to search around for error handling if you hit an unexpected error in yours or someone else's code.

**Review by elijaharley — APPROVED** [review]
!!Nice work with this Linda, I've left a couple of comments for you to consider but you're good to merge.

File: __tests__/app.test.js (hunk 1 above)
Comment (xViolaine) [reply __tests__/app.test.js:188]: no, it's the poor man's type cast 🤣

File: __tests__/app.test.js (hunk 2 above)
Comment (xViolaine) [reply __tests__/app.test.js]: Thanks! I've gotten so many tips now on how to best organize my error handling and testing in general, I'm not even sure anymore how to implement them all. :D I thought this was a nice and concise way of handling things, but I'll give it a switch around and see how I fare!

**Pull request merged**